openstf/model/serializer.py

```diff
@@ -0,0 +1,204 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+from abc import ABC, abstractmethod
+from datetime import datetime
+from pathlib import Path
+
+import joblib
+import structlog
+from sklearn.base import RegressorMixin
+
+from ktpbase.config.config import ConfigManager
+
+
+MODEL_FILENAME = "model.sav"
```

Comment:
Is this a common convention? Its maybe not that important, but a convention would be good I guess. And its not something you can easily change after this is merged. Once the models are stored using a certain extension you need extra code to handle a migration.

Reply:
sklearn uses .joblib in their examples, I suggest we use that one.

Reply:
Sound good!